listings/models.py

```diff
@@ -1,3 +1,15 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Listing(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Thanks for the comment Jay. The `id` field is added automatically to all models, according to https://docs.djangoproject.com/en/1.9/topics/db/models/